homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -56,32 +57,51 @@ def should_poll(self):
     @property
     def address(self):
         """Return the address of the node."""
-        return self._address
+        return self._insteon_device.address.human
 
     @property
-    def name(self):
-        """Return the name of the node."""
-        return self._name
+    def group(self):
```

Comment:
Where is this used?

Reply:
It is being used in the state machine attributes. I removed the manufacturer, model number, etc per you request above and only left two items, address and group. These are the minimum attributes to describe the unique entity of the physical device.